src/test/java/org/apache/tomcat/jakartaee/ManifestConverterTest.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.tomcat.jakartaee;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class ManifestConverterTest {
+
+	@Test
+    public void testAccepts() {
```

Comment:
Note the code format.

Reply:
I pushed an additional commit that fixes the whitespace formatting. Does it address this comment?